tests/components/russound_rio/conftest.py

```diff
@@ -0,0 +1,127 @@
+"""Test fixtures for Russound RIO integration."""
+
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+from homeassistant.components.russound_rio.config_flow import FlowHandler
+from homeassistant.components.russound_rio.const import DOMAIN
+from homeassistant.components.russound_rio.media_player import RussoundZoneDevice
+from homeassistant.config_entries import ConfigEntryState
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+
+MOCK_CONFIG = {
+    "host": "127.0.0.1",
+    "port": 9621,
+}
+
+MOCK_DATA = {
+    "host": "127.0.0.1",
+    "port": 9621,
+    "model": "MCA-C5",
+}
+
+
+@pytest.fixture(autouse=True)
+def bypass_setup_fixture():
+    """Prevent setup."""
+    with patch(
+        "homeassistant.components.russound_rio.async_setup_entry", return_value=True
+    ):
+        yield
+
+
+@pytest.fixture(name="mock_config_entry")
+def mock_config_entry_fixture():
+    """Create a mock config entry."""
+    return MockConfigEntry(
+        domain=DOMAIN, data=MOCK_CONFIG, state=ConfigEntryState.LOADED
+    )
+
+
+@pytest.fixture(name="mock_russound")
+def mock_russound_fixture():
+    """Mock the Russound RIO library."""
+    with patch(
+        "homeassistant.components.russound_rio.Russound", autospec=True
+    ) as russound_mock:
+        russound_mock.enumerate_controllers.return_value = [
+            (1, "00:11:22:33:44:55", "MCA-C5")
+        ]
+        yield russound_mock
+
+
+@pytest.fixture(name="mock_russound_alt")
+def mock_russound_alt_fixture():
+    """Mock the Russound RIO library."""
+    with patch("aiorussound.Russound", autospec=True) as russound_mock:
+        yield russound_mock
+
+
+@pytest.fixture
+def mock_asyncio_timeout():
+    """Mock the asyncio timeout."""
+    with patch("homeassistant.components.russound_rio.asyncio.timeout") as timeout_mock:
+        yield timeout_mock
```

Comment:
Let's replace the `5` in the timeout with a constant, and then we can patch that constant to be `0`.

Reply:
This ended up being unused